packages/vitest-environment-nuxt/src/index.ts

```diff
@@ -53,13 +54,20 @@ export default <Environment>{
     app.id = environmentOptions.nuxt.rootId
     win.document.body.appendChild(app)
 
-    win.IntersectionObserver =
-      win.IntersectionObserver ||
-      class IntersectionObserver {
-        observe() {}
-        unobserve() {}
-        disconnect() {}
-      }
+    if (environmentOptions?.nuxt?.mock?.intersectionObserver) {
```

Comment:
We should probably default this to `true` so as not to break existing setups.

```suggestion
    if (environmentOptions?.nuxt?.mock?.intersectionObserver !== false) {
```